tests/ConfigTest.php

```diff
@@ -175,8 +177,8 @@ public function test_it_is_foreachable()
             'baz' => true
         ]);
 
-        foreach ($config as $item) {
-            $this->assertTrue($item);
+        foreach ($config->toArray() as $item => $value) {
+            $this->assertTrue($value);
```

Comment:
Thanks for this PR but this test was specifically checking if the Config object itself is foreachable. By using `$config->toArray()` this test no longer tests that. Also the `=> $value` is unnecessary. We don't care about the index in this test so we can just leave it as `foreach ($config as $item)`.